universal-application-tool-0.0.1/app/controllers/dev/DevController.java

```diff
@@ -19,7 +19,7 @@ public DevController(Environment environment, Config configuration) {
   }
 
   public boolean isDevEnvironment() {
-    if (environment.isDev()) {
+    if (environment.isDev() || environment.isTest()) {
```

Comment:
Thoughts: I think we might not need this, the only test that touches this set its environment to dev already.

Reply:
Yeah, those tests are currently giving me trouble ... suppose I could try another tack.